_includes/cloud-table/cloud-summary-table-pk.md

```diff
@@ -1,6 +1,6 @@
-The table ID column (`_id`) is used as the table primary key and must uniquely represent a record in your table.
+The `_id` column in each table represents the primary key.
 
-| UI Type | SQL data type | Data source table unique key |
-|---|---|---|
-| Number | [`id`](/docs/sql-guide/data-types/data-type-id) | Positive integer |
-| String | [`string`](/docs/sql-guide/data-types/data-type-string) | String key or a combination of columns used to create a unique key |
+| Intended key | UI type | SQL data type | Disk utilization (no data) | Additional information |
```

Comment:
I don't really understand what this is trying to convey. "Intended key" doesn't make sense to me.

Is this trying to explain the two types of primary key and the pros/cons of each?

In that case, the important thing to convey is that if you use the `ID` type your primary key is an integer, and you want them to be mostly contiguous (7,8,9,10,11,12...). If you use random integers for your `ID` type primary key, it's disastrous for performance and storage size. Several large blocks of contiguous numbers that are separated is also fine.

String primary keys are automatically mapped to blocks of contiguous integers internally. Storing that mapping costs performance and storage, but saves the user from having to manage handing out contiguous integer IDs.

Reply:
All great information I wasn't aware of until now.
I've rejigged the table with your paraphrased information, and it now appears to work much better :D